Mux-Stats-Kaltura.podspec

```diff
@@ -26,6 +26,6 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '9.0'
   s.tvos.vendored_frameworks = 'XCFramework/MUXSDKKaltura.xcframework'
 
-  s.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386' }
+  s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
```

Comment:
will this not be overwritten by the following line?

Reply:
oh I think you're right. I didn't know how this works but you're right we probably need to put all the settings in the same line. Yeah... I'll update, thanks.

Reply:
updated